### PR TITLE
Setup a device with Ignition

### PR DESCRIPTION
**Pull request by mmartinv**
- **bug: fix bad document formatting**
- **user-docs: use a dedicated document for Simplified Installer**
- **user-docs: document how to use Simplified Provisioner and Ignition**

Fixes: fedora-iot/iot-distro#29

### CODE REVIEW

File: user-docs/modules/ROOT/pages/booting-the-simplified-installer.adoc [hunk 1]
```diff
@@ -0,0 +1,147 @@
+include::_attributes.adoc[]
+
+= Boot the Simplified Installer
```
Comment (miabbott) [inline user-docs/modules/ROOT/pages/booting-the-simplified-installer.adoc:3]: Are we sticking with "simplified installer" or should we start using "simplified provisioner"?

@pcdubs @nullr0ute

Comment (nullr0ute) [reply user-docs/modules/ROOT/pages/booting-the-simplified-installer.adoc:3]: The later, so "simplified provisioner"

Comment (mmartinv) [reply user-docs/modules/ROOT/pages/booting-the-simplified-installer.adoc]: done

**Review by nullr0ute — APPROVED** [review]
LGTM

**Pull request merged**